### DISCUSSION TITLE
Is it possible to replace the secret key?

### DISCUSSION THREAD

**Discussion by joubbi** [netbox-community/netbox#8183] [Q&A] [answered]
I have an installation of Netbox running fine.
The `SECRET_KEY` in `configuration.py` is not so secret anymore.
Is it possible to replace the key with something new without losing data?

**Comment by kkthxbye-code** [comment, marked as answer]
https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-SECRET_KEY

Sessions will be invalidated, that's pretty much it.

**Reply by jeremystretch** [reply]
Yep; rotating the secret key will invalidate all existing user cookies, but it isn't tied to any actual data.

**Reply by joubbi** [reply]
Thanks!
I think that the significance (or lack of) the secret key would be good information to have in the documentation.

I found this (the update on the first post), which helped me migrate an installation better than the official documentation:
https://www.reddit.com/r/networking/comments/ms5p2e/netbox_import_database_on_new_install/
But the part about the secret key got me wondering.
